unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/AudioSource/AudioSourceRegister.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using DCL.ECSRuntime;
+using DCL.SettingsCommon;
+
+namespace DCL.ECSComponents
+{
+    public class AudioSourceRegister : IDisposable
```

Comment:
I see all the registers are basically identical.

Maybe we can have a `DefaultRegister<THandler>` to avoid boilerplate?
We could still create a custom register if any of the components need specific stuff.

Reply:
I thought to make a single code to do it, but this way we can encapsulate assembly. Since each handler has different parameters in the constructor.

I can go for that if you want!